lib/Home.dart

```diff
@@ -51,7 +50,6 @@ class _HomeState extends State<Home> {
   static const int PROFILE = 2;
 
   int _selectedIndex = RIDES;
-  List<FutureRide> rides;
```

Comment:
Was this simply not used before? This isn't a requested change lol. Just hard to scroll through the unchanged code and check.

Reply:
This was from the previous homepage design, I missed it for the last PR when we changed the homepage for the new cards. On that note, I think we could actually remove PastRide and Upcoming since those are from previous designs and we don't use them anywhere anymore?

Reply:
I'd just confirm with the designers that those (or something similar) won't be coming back. Then, if they aren't used anywhere anymore, I agree with you that they should be removed.